<launch>
    <node pkg="tinker_camera" type="com_arm_cam.py" name="com_arm_cam" output="screen" args="1"/>
    <node pkg="tinker_object_recognition" type="collect_sample_node" name="collect_sample_node" output="screen">
        <param name="filepath" value="$(find tinker_object_recognition)/img/"/>
        <rosparam command="load" file="$(find tinker_object_recognition)/param/image_info_new.yaml"/>
        <rosparam command="load" file="$(find tinker_object_recognition)/param/fd_param.yaml"/>
    </node>
</launch>
